packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -135,140 +138,111 @@ function Root({
   )
 
   return (
-    <div
-      className={clsx(
-        styles['SubdomainNavBar-outer-container'],
-        fixed && styles['SubdomainNavBar-outer-container--fixed'],
-      )}
-    >
-      <header className={clsx(styles['SubdomainNavBar'], className)} data-testid={testIds.root} {...rest}>
-        <div
-          ref={focusTrapRef}
-          className={clsx(
-            styles['SubdomainNavBar-inner-container'],
-            searchVisible && styles['SubdomainNavBar-inner-container--search-open'],
-            !fullWidth && styles['SubdomainNavBar-inner-container--centered'],
-          )}
-          data-testid={testIds.innerContainer}
-        >
-          <nav aria-label="global breadcrumb">
-            <ol className={styles['SubdomainNavBar-title-area']}>
-              <li>
-                <a href={logoHref} aria-label="Github Home" className={styles['SubdomainNavBar-logo-mark']}>
-                  <span className={clsx(styles['SubdomainNavBar-back-arrow'])}>
-                    <ChevronLeftIcon fill="currentColor" size={24} />
-                  </span>
-                  <MarkGithubIcon fill="currentColor" size={24} />
-                </a>
-              </li>
-              {title && isSmall && (
-                <>
-                  <li role="separator" className={styles['SubdomainNavBar-title-separator']} aria-hidden>
-                    /
-                  </li>
-                  <li>
-                    <a href={titleHref} aria-label={`${title} home`} className={clsx(styles['SubdomainNavBar-title'])}>
-                      {title}
-                    </a>
-                  </li>
-                </>
-              )}
-            </ol>
-          </nav>
-          {hasLinks && (
-            <nav
-              id="menu-navigation"
-              aria-label={title}
-              className={styles['SubdomainNavBar-primary-nav']}
-              data-testid={testIds.menuLinks}
-            >
-              <NavigationVisbilityObserver className={clsx(styles['SubdomainNavBar-primary-nav-list--invisible'])}>
-                {menuItems}
-              </NavigationVisbilityObserver>
+    <>
+      <Button
+        as="a"
+        href={`#${startOfContentID}`}
+        variant="primary"
+        className={clsx(styles['SubdomainNavBar-skip-to-content'], !startOfContentButtonFocused && 'visually-hidden')}
+        onFocus={() => setStartOfContentButtonFocused(true)}
+        onBlur={() => setStartOfContentButtonFocused(false)}
```

Comment:
Possible to extract these functions outside of the return statement please? Potentially wrapping in a `useCallback`? Micro-perf optimisation to avoid creating a new function each time. I missed this last time, sorry.